sitecore/9.0.1 rev. 171219 XM1/VS2017/Website/README.md

```diff
@@ -0,0 +1,101 @@
+# docker-demo for Sitecore 9.0.1 rev. 171219 XM1
+
+This demo showcases how to create a VS2017 project for Sitecore with the following features:
+
+1. Website folder as volume with files stored in the development machine, at debugging time (no need for synchronization)
+1. Website folder embedded in the image at runtime.
+1. Website folder files, databases and Solr cores are automatically copied from the images to the Development environment (if missing) (same as databases).
+1. Attach to IIS process in the container from VS2017 for debugging
+
+## Prerequisites
+
+* Windows 10 Fall Creators Update
+* Docker for Windows
+* Visual Studio 2017 15.5 or later
+* Clone the [Sitecore base docker-images repository](https://github.com/Sitecore/docker-images) and build the images for the Sitecore version "9.0.1 rev. 171219 XM1 CM".
+
+## Development Environment Setup
+
+1. Clone [this repository](https://github.com/Ben-m-s/docker-demo) into a folder such as �c:\Docker\Sitecore\docker-demo� (next steps will asume this folder has been used):
```

Comment:
Shouldn't it be `https://github.com/Sitecore/docker-demo`?

Reply:
Sure. It should. The same way that all paths will need to change in order to accommodate naming changes being currently discussed in Slack. I used that temporary URL in order to facilitate the testing before the pull takes place.

Thanks for flagging it up. It will work as a reminder.

Please let me know whether is better to change it already.

Reply:
Agreed - If it's a PR to Sitecore/docker-demo but the README points to Ben-m-s/docker-demo I'd say it should be changed before getting merged

Reply:
The URL has been updated in the README file. :-)